about/complying_with_licenses.rst

```diff
@@ -123,71 +121,44 @@ The Godot Engine developers consider that a link to ``godotengine.org/license``
+    - :ref:`Engine.get_license_text<class_Engine_method_get_license_text>`
 
-    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+    For third-party components used by the engine:
 
-mbed TLS
-^^^^^^^^
+    - :ref:`Engine.get_license_info<class_Engine_method_get_license_info>`
+    - :ref:`Engine.get_copyright_info<class_Engine_method_get_copyright_info>`
 
-If the project is exported with Godot 3.1 or later, it includes `mbed TLS <https://tls.mbed.org>`_.
-The Apache license needs to be complied to by including the following text:
+.. _doc_complying_with_licenses_thirdparty:
 
-    Copyright The Mbed TLS Contributors
+Third-party licenses
+--------------------
 
-    Licensed under the Apache License, Version 2.0 (the "License"); you may
-    not use this file except in compliance with the License.
-    You may obtain a copy of the License at
+Godot itself contains software written by
+`third parties <https://github.com/godotengine/godot/blob/master/thirdparty/README.md>`_,
+which is compatible with, but not covered by Godot's MIT license.
 
-    http://www.apache.org/licenses/LICENSE-2.0
+Many of these dependencies are distributed under permissive open source licenses
+which require attribution by explicitly citing their copyright statement and
+license text in the final product's documentation.
 
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
+Given the scope of the Godot project, this is fairly difficult to do thoroughly.
+For the Godot editor, the full documentation of third-party copyrights and
+licenses is provided in the `COPYRIGHT.txt <https://github.com/godotengine/godot/blob/master/COPYRIGHT.txt>`_
+file.
 
-.. note::
+A good option for end users to document third-party licenses is to include this
+file in your project's distribution, which you can e.g. rename to
+``GODOT_COPYRIGHT.txt`` to prevent any confusion with your own code and assets.
 
-    If you exported your project using a
-    :ref:`custom build with specific modules disabled <doc_optimizing_for_size>`,
-    you don't need to list the disabled modules' licenses in your exported project.
+If you export your project using a
+:ref:`custom build with specific modules disabled <doc_optimizing_for_size>`,
+you can remove the copyright lines for the libraries you compiled out.
```

Comment:
```suggestion
you can remove the copyright lines for the libraries you disabled.
```
I can't make sense of "compiled out" as a phrase

Reply:
I removed the paragraph as a whole in the end.